internal/cli/options.go

```diff
@@ -83,13 +91,26 @@ func appConfiguration(cmd *cobra.Command) (models.ApplicationUpdateRequest, erro
 	if instances != nil {
 		result.Instances = instances
 	}
-	// nil --> Default / No change
-	// - AppCreate API will replace it with `v1.DefaultInstances`
-	// - AppUpdate API will treat it as no op, i.e. keep current instances.
 
 	result.Services = uniqueStrings(services)
 	sort.Strings(result.Services)
 
+	assignments, err := cmd.Flags().GetStringSlice("env")
+	if err != nil {
+		return result, errors.Wrap(err, "failed to read option --env")
+	}
+
+	for _, assignment := range assignments {
+		pieces := strings.Split(assignment, "=")
+		if len(pieces) != 2 {
+			return result, errors.New("Bad --env assignment, expected `name=value` as value")
```

Comment:
Maybe add the `assignment` value to the error message to show which `--env` assignment is meant here ?!